core/pom.xml

```diff
@@ -40,6 +40,11 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
+      <groupId>com.alibaba</groupId>
+      <artifactId>fastjson</artifactId>
```

Comment:
Is this used for sending and receiving dictionary between client and server?
JSON is text based, but in this use case I think it is better to use a binary based protocol, can we use a more simpler and compact one?